If line_profiler is not located in the default `sys.path` when those two test cases are run, then they will fail:

```
[   92s] _____________ test_explicit_profile_ignores_inherited_owner_marker _____________
[   92s] 
[   92s]     def test_explicit_profile_ignores_inherited_owner_marker():
[   92s]         """
[   92s]         Standalone runs should not be blocked by an inherited owner marker.
[   92s]         """
[   92s]         with tempfile.TemporaryDirectory() as tmp:
[   92s]             temp_dpath = ub.Path(tmp)
[   92s]             env = os.environ.copy()
[   92s]             env['LINE_PROFILE'] = '1'
[   92s]             env['LINE_PROFILER_OWNER_PID'] = str(os.getpid() + 100000)
[   92s]             env['PYTHONPATH'] = os.getcwd()
[   92s]     
[   92s]             with ub.ChDir(temp_dpath):
[   92s]                 script_fpath = ub.Path('script.py')
[   92s]                 script_fpath.write_text(_demo_explicit_profile_script())
[   92s]     
[   92s]                 args = [sys.executable, os.fspath(script_fpath)]
[   92s]                 proc = ub.cmd(args, env=env)
[   92s]                 print(proc.stdout)
[   92s]                 print(proc.stderr)
[   92s] >               proc.check_returncode()
[   92s] 
[   92s] tests/test_explicit_profile.py:166: 
[   92s] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[   92s] 
[   92s] self = {'out': '', 'err': 'Traceback (most recent call last):\n  File "/tmp/tmpwrpbh7p8/script.py", line 1, in <module>\n    ...en: returncode: 1 args: ['/usr/bin/python3.11', 'script.py']>, 'cwd': None, 'command': '/usr/bin/python3.11 script.py'}
[   92s] 
[   92s]     def check_returncode(self) -> None:
[   92s]         """Raise CalledProcessError if the exit code is non-zero."""
[   92s]         import subprocess
[   92s]     
[   92s]         if self.returncode:
[   92s] >           raise subprocess.CalledProcessError(
[   92s]                 self.returncode, self.args, self.stdout, self.stderr
[   92s]             )
[   92s] E           subprocess.CalledProcessError: Command '['/usr/bin/python3.11', 'script.py']' returned non-zero exit status 1.
[   92s] 
[   92s] /usr/lib/python3.11/site-packages/ubelt/util_cmd.py:174: CalledProcessError
[   92s] ----------------------------- Captured stdout call -----------------------------
[   92s] 
[   92s] Traceback (most recent call last):
[   92s]   File "/tmp/tmpwrpbh7p8/script.py", line 1, in <module>
[   92s]     from line_profiler import profile
[   92s] ModuleNotFoundError: No module named 'line_profiler'
[   92s] 

```

Our build environment has the module installed in a path that isn't included in `sys.path` by default. I'm happy to modify the PR if you do wish to proceed.

--- a/tests/test_explicit_profile.py
+++ b/tests/test_explicit_profile.py
@@ -153,7 +153,9 @@ def test_explicit_profile_ignores_inherited_owner_marker():
         env = os.environ.copy()
         env['LINE_PROFILE'] = '1'
         env['LINE_PROFILER_OWNER_PID'] = str(os.getpid() + 100000)
-        env['PYTHONPATH'] = os.getcwd()
+        env['PYTHONPATH'] = os.pathsep.join(
+            [p for p in [os.getcwd(), env.get('PYTHONPATH')] if p]
+        )
 
         with ub.ChDir(temp_dpath):
             script_fpath = ub.Path('script.py')
@@ -182,7 +184,9 @@ def test_explicit_profile_process_pool_forkserver():
         env = os.environ.copy()
         env['LINE_PROFILE'] = '1'
         # env['LINE_PROFILER_DEBUG'] = '1'
-        env['PYTHONPATH'] = os.getcwd()
+        env['PYTHONPATH'] = os.pathsep.join(
+            [p for p in [os.getcwd(), env.get('PYTHONPATH')] if p]
+        )
 
         with ub.ChDir(temp_dpath):
             script_fpath = ub.Path('script.py')

